projects/fsastorefrontlib/src/occ/services/default-occ-messages-config.ts

```diff
@@ -0,0 +1,12 @@
+import { FSOccConfig } from '../config/fs-occ-config';
+
+export const occMessagesConfig: FSOccConfig = {
```

Comment:
No need for this file. there is already defaultOccInboxConfig which is used for inbox and site messages.
Move numberOfUnreadMessages endpoint to that file instead.